frontend/app/src/components/AssetBalances.vue

```diff
@@ -153,7 +155,7 @@ const sortItems = getSortItems(asset => get(assetInfo(asset)));
     </template>
     <template #item.expand="{ item }">
       <row-expander
-        v-if="item.breakdown || isEvmNativeToken(item.asset)"
+        v-if="!hideBreakdown && (item.breakdown || isEvmNativeToken(item.asset))"
```

Comment:
@lukicenturi can you run `pnpm run lint:fix` here? it seems like a fixable thing.